http/events_test.go

```diff
@@ -234,32 +238,39 @@ func TestEventsSubscribeNamespaces(t *testing.T) {
 			t.Cleanup(func() {
 				conn.Close(websocket.StatusNormalClosure, "")
 			})
-			sendEvents()
+			err = sendEvents()
+			if err != nil {
+				t.Fatal(err)
+			}
 			// CI is sometimes slow, so this timeout is high initially
-			ctx, cancel := context.WithTimeout(ctx, 1*time.Second)
-			defer cancel()
+			timeout := 10 * time.Second
 			gotEvents := 0
 			for {
+				ctx, cancel := context.WithTimeout(ctx, timeout)
+				defer cancel()
```

Comment:
Even though I know this is okay, my IDE doesn't like it (`defer` in a for-loop), so I usually do `t.Cleanup(cancel)`.

Reply:
I'll update this! Thanks @swenson